tests/strategies/external/test_init.py

```diff
@@ -583,3 +585,38 @@ def test_set_energy_measurement_impl_fails_for_negative_energy(self, ext_strateg
                                "status": "error",
                                "transaction_id": arguments["transaction_id"],
                                "error_message": error_message})
+
+    @pytest.mark.parametrize(
+        "ext_strategy, command_name",
+        itertools.product(
+            [LoadForecastExternalStrategy(), PVForecastExternalStrategy()],
+            ["set_energy_forecast", "set_energy_measurement"]
+        )
+    )
```

Comment:
Pytest now supports cartesian product out of the box (by stacking decorators):
```suggestion
    @pytest.mark.parametrize(
        "ext_strategy", [LoadForecastExternalStrategy(), PVForecastExternalStrategy()])
    @pytest.mark.parametrize("command_name", ["set_energy_forecast", "set_energy_measurement"])
```

Reply:
Did not know that, thanks!